codespell_lib/data/dictionary.txt

```diff
@@ -13845,6 +13854,7 @@ geometrys->geometries
 geomety->geometry
 geometyr->geometry
 geomitrically->geometrically
+geomoetric->geometric
```

Comment:
Can you add the -try and -trically versions of this typo too please.

Reply:
added.